quipucords/quipucords/settings.py

```diff
@@ -283,6 +283,8 @@
     },
 }
 
+INPUT_LOG_BASENAME = os.path.join(BASE_DIR, 'input_log')
```

Comment:
I think this should go into the specified logs directory when running in production.
I also think we need to improve the name for the variable and log its self, as it is not very descriptive. Can we also use `.log` extension instead of underscore? This will help with auto suggest tools that read logs.

Is there any mechanism to toggle this function ON/OFF via environment variable?

Reply:
I'll move the logs to the right directory as soon as we agree on a name for them. :)